### PR TITLE
[TACHYON-1562] Remove raw tables

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1562

### CODE REVIEW

**Comment by jsimsa** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

File: common/src/main/resources/tachyon-default.properties [hunk 1]
```diff
@@ -148,7 +148,6 @@ tachyon.user.lineage.master.client.threads=10
 tachyon.user.network.netty.timeout.ms=3000
 tachyon.user.network.netty.worker.threads=0
 tachyon.user.quota.unit.bytes=8MB
-tachyon.user.raw.table.master.client.threads=10
```
Comment (apc999) [inline common/src/main/resources/tachyon-default.properties]: hwo about these two properties:
```
tachyon.max.columns=1000
tachyon.max.table.metadata.bytes=5MB
```

**Comment by apc999** [comment]
left one comment. Otherwise it looks good to me

File: common/src/main/resources/tachyon-default.properties (hunk 1 above)
Comment (aaudiber) [inline common/src/main/resources/tachyon-default.properties]: Good call, removed

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**